app/assets/stylesheets/application.css

```diff
@@ -10,90 +10,9 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
  *= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
  *= require_self
```

Comment:
In order to ensure the precedence always stays the same, it might make sense to be explicit about the two new files:
```
  *= require_self
  *= require site.scss
  *= require datatables.css.scss
  *= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
  *= require_tree.